packages/grid/x-data-grid/src/components/panel/GridPanelFooter.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
 import clsx from 'clsx';
-import { styled, SxProps, Theme } from '@mui/material/styles';
+import { styled, SxProps, Theme } from '@mui/system';
```

Comment:
We discussed it recently with @LukasTy 
From what I saw, the `styled` from `@mui/material` have a slightly different behavior (with a different default them for instance).

Did we check with the core that this would not cause a problem ?
For components with no usage of `theme` I guess it's fine.